app-misc/asciinema/asciinema-2.0.1.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="Terminal session recorder"
```

Comment:
I'm not sure about this, but looking at what we have in most recipes I think this might be preferred here:
```
-SUMMARY="Terminal session recorder"
+SUMMARY="A terminal session recorder"
```
That said, I'm fine with the current summary too.

Reply:
BTW, maybe we should skip some files in the installation:
```diff
-documentation/packages/asciinema/CHANGELOG.md
-documentation/packages/asciinema/CODE_OF_CONDUCT.md
-documentation/packages/asciinema/CONTRIBUTING.md
 documentation/packages/asciinema/README.md
 documentation/packages/asciinema/asciicast-v1.md
 documentation/packages/asciinema/asciicast-v2.md
```
(Maybe we can even drop the last two files, `asciicast-v*.md`. The readme file has pointers to the project's repos on GitHub so can probably drop all but that readme file. What do you think?)

Reply:
> BTW, maybe we should skip some files in the installation:
> 
> ```diff
> -documentation/packages/asciinema/CHANGELOG.md
> -documentation/packages/asciinema/CODE_OF_CONDUCT.md
> -documentation/packages/asciinema/CONTRIBUTING.md
>  documentation/packages/asciinema/README.md
>  documentation/packages/asciinema/asciicast-v1.md
>  documentation/packages/asciinema/asciicast-v2.md
> ```
> 
> (Maybe we can even drop the last two files, `asciicast-v*.md`. The readme file has pointers to the project's repos on GitHub so can probably drop all but that readme file. What do you think?)

Personally, I'd keep `asciicast-v*.md`, `CHANGELOG.md`, and `README.md`, but I don't have much experience or knowledge in this; I'll be fine with whatever you decide.